docs/en/latest/terminology/global-rule.md

```diff
@@ -32,6 +32,12 @@ A [Plugin](./plugin.md) configuration can be bound directly to a [Route](./route
 
 Compared with the plugin configuration in Route, Service, Plugin Config, and Consumer, the plugin in the Global Rules is always executed first.
 
+::: note
+
+For stream routes, only plugins that run on `before_proxy` phase get executed as Global Rules.
```

Comment:
I think this is a quite in-depth note for new users, e.g.

* What are phases? What is the `before_proxy` phase? Why should I care about phases when I want to apply a plugin globally when using APISIX as a L4 stream proxy? (leaky abstraction?)
* What plugins work for global rules on L4? What don't? Does `prometheus` work? Does `syslog` work?

Reply:
```suggestion
For stream routes, only plugins that run on `before_proxy` [phase](../terminology/plugin.md#plugins-execution-order) get executed as Global Rules.
```